Source_localization.py

```diff
@@ -0,0 +1,335 @@
+# -*- coding: utf-8 -*-
+#######################################################
+#                                                     #
+# small utility function to handle file lists         #
+#                                                     #
+#######################################################
+def get_files_from_list(fin):
```

Comment:
This function is repeated many times and we also have it in jumeg_base (will be moved to jumeg_utils soon). So it can be removed from both scripts.